sqldelight-gradle-plugin/src/test/fixtures/function-name-tests/src/main/sqldelight/com/sample/Capitals.sq

```diff
@@ -0,0 +1,3 @@
+capitals_work_fine:
+SELECT COUNT(*), MAX(some_int)
+FROM test;
```

Comment:
I don't think it's set up that way. I just want to make sure the next time someone re-generates the files there's not a new one being generated under this test.